lib/db/schema/patch-045-046.sql

```diff
@@ -0,0 +1,224 @@
+-- Add emails table and corresponding procedures to
+-- create, get and delete them.
+
+CREATE TABLE IF NOT EXISTS emails (
+  normalizedEmail VARCHAR(255) NOT NULL PRIMARY KEY,
```

Comment:
Since we're creating a table of emails, this seems like an excellent time to make the primary key something smaller, like an auto incrementing integer. We've talked about eventually having the `accounts` and now `emailBounces` tables hold a foreign key to an `emails` table, to improve lookup times, and reduce the number of times we have the literal email text duplicated through the database.